en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md

```diff
@@ -1,175 +1,213 @@
+
+4. Set the new keystore details when you change the keystore.
+
+      1. Navigate to the <ANALYTICS_HOME>/wso2/dashboard/bin/carbon.sh` file.
       
-    | Element                | Description                                                      |
-    |------------------------|------------------------------------------------------------------|
-    | **privateKeyAlias**    | Private key password                                                |
-    | **keyStoreLocation**   | Name of the keystore file. **The file must be located in `<API-M_ANALYTICS_HOME>/resources/security`** |
-   
+      2. Define the correct keystore path for the `Djavax.net.ssl.keyStore` system property.
+      
+      3. Define the correct keystore password for the `Djavax.net.ssl.keyStorePassword` system property.
+
+## Configuring keystores in worker profile
 
-## Configuring Keystores in Worker Profile
+!!! note
+    Other than explicitly mentioned, you need to do all the following configuration updates related to the worker profile in the `<API-M_ANALYTICS_HOME>/conf/worker/deployment.yaml` file.
 
-For the dashboard profile the required updates need to be done to the `<API-M_ANALYTICS_HOME>/conf/worker/deployment.yaml` file for following use cases.
+1. Update the databridge configurations.
 
-1. **Databridge configurations**
+    This is used for authenticating communication over SSL/TLS between API Manager and API-M Analytics. Handles thrift and binary transports.
 
-    This is used for authenticating communication over SSL/TLS between API Manager and APIM Analytics. Handles thrift and binary transports.
     ```yaml
-       databridge.config:
-         ... 
-         keyStoreLocation : ${sys:carbon.home}/resources/security/wso2carbon.jks
-         keyStorePassword : wso2carbon
-         ... 
+    databridge.config:
+      ... 
+      keyStoreLocation : ${sys:carbon.home}/resources/security/wso2carbon.jks
+      keyStorePassword : wso2carbon
+      ... 
     ```
     The elements in the above configuration are described below:
    
     | Element                | Description                                                      |
     |------------------------|------------------------------------------------------------------|
-    | **keyStoreLocation**   | Name of the keystore file. **The file must be located in `<API-M_ANALYTICS_HOME>/resources/security`** |
+    | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
     | **keyStorePassword**   | Keystore password                                                |
 
-2. **HTTP transport configuration**
+2. Update the HTTP transport configurations.
 
-    This is used for authentication, encryption and signing for the analytics dashboard web application.
+    This is used for authentication, encryption, and signing for the analytics dashboard web application.
```

Comment:
```suggestion
    This is used for authentication, encryption, and signing for the analytics worker.
```

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1231/commits/f872500a49ef0c6223fab16659a66f2e133faa22